File: app/src/main/java/com/eam/investire/helper/Base64Custom.java
package com.eam.investire.helper;

import android.util.Base64;

public class Base64Custom {

    public static String codificarBase64(String texto) {
        return Base64.encodeToString(texto.getBytes(), Base64.DEFAULT).replaceAll("([\\n\\r])", "");
    }

}

File: app/src/main/java/com/eam/investire/helper/Base64Custom.kt
package com.eam.investire.helper

import android.util.Base64

object Base64Custom {
    @JvmStatic
    fun codificarBase64(texto: String): String {
        return Base64.encodeToString(texto.toByteArray(), Base64.DEFAULT).replace("([\\n\\r])".toRegex(), "")
    }
}